packagelist/debian_12_headless.txt

```diff
@@ -73,10 +83,24 @@ autoconf
 automake
 build-essential
 command-not-found
+dialog
 gdb
 git
+initramfs-tools
+libgpiod-dev
 libiio-dev
+libiio0-doc
 libubootenv-tool
 lsof
 meson
 nfs-common
+sqlite3
+xxd
+
+# Development tools - C/C++
+
+# Development tools - Python
+python3-bluez
+python3-can
+python3-libiio
+python3-pip
```

Comment:
So far we haven't been installing many of the python modules.  Roughly how much space does this add?  If its not much then I'd agree, but if the disk space usage makes sense I think we should add libgpiod's python bindings to the default list .

Reply:
To the compressed rootfs, these add around 30 MB (from 251 MB to 285 MB for the "X11" image).

I agree, it would be good for `python3-libgpiod` to be there too.

Reply:
I'd be fine with adding that in then.  Make sure its common among the headless images, including the ubuntu variants though.

Reply:
Done.